front/src/applications/editor/tools/switchEdition/components/TrackNodeTypeDiagram.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { TRACK_NODE_TYPES_ID, TrackNodeTypeId } from 'types';
+import link from 'assets/pictures/nodeTypes/link.svg';
+import crossing from 'assets/pictures/nodeTypes/crossing.svg';
+import pointSwitch from 'assets/pictures/nodeTypes/point_switch.svg';
+import singleSlipSwitch from 'assets/pictures/nodeTypes/single_slip_switch.svg';
+import doubleSlipSwitch from 'assets/pictures/nodeTypes/double_slip_switch.svg';
```

Comment:
I think you forgot to rename the folder or to change the path to your images here
```suggestion
import link from 'assets/pictures/trackNodeTypes/link.svg';
import crossing from 'assets/pictures/trackNodeTypes/crossing.svg';
import pointSwitch from 'assets/pictures/trackNodeTypes/point_switch.svg';
import singleSlipSwitch from 'assets/pictures/trackNodeTypes/single_slip_switch.svg';
import doubleSlipSwitch from 'assets/pictures/trackNodeTypes/double_slip_switch.svg';
```

Reply:
Oups my bad. Done :)